Tools/CISupport/ews-app/ews/common/github.py

```diff
@@ -204,7 +204,7 @@ class GitHubEWS(GitHub):
                           ['bindings', 'ios-sim', 'mac-debug', 'gtk', 'wincairo'],
                           ['webkitperl', 'ios-wk2', 'mac-AS-debug', 'gtk-wk2', ''],
                           ['webkitpy', 'api-ios', 'api-mac', 'api-gtk', ''],
-                          ['jsc', 'tv', 'mac-wk1', 'jsc-armv7', ''],
+                          ['jsc', 'tv', 'mac-wk1', 'jsc-armv7', 'JSC-Tests-arm64-EWS', ''],
```

Comment:
I think this should be named `jsc-arm64`, but also I think we might need to make a new list because each line represents a row, and each item in the list represents a column in the table @aj062